pkg/util/selector_test.go

```diff
@@ -310,7 +312,7 @@ func TestClusterMatches(t *testing.T) {
 		want bool
 	}{
 		{
-			name: "cluster excluded",
+			name: "[case 1] cluster excluded",
```

Comment:
I don't think we need to organize the tests by an explicit `index`, that is not useful but brings extra maintenance costs.

Can we organize the cases in the order of fields that are present in the struct? Like:
```
- test label selector
- test field selector
- test cluster names
- test exclude clusters
```

Reply:
Modified, no major changes have been made to the existing tests. Added a field selector test after the tests. In order to ensure that the test is quickly located to a certain test by the test name in case of a test error, I described the test name in as much detail as possible as you suggested